src/editor/api/auth.js

```diff
@@ -14,8 +21,55 @@ const signIn = async () => {
       name: user.displayName
     });
   } catch (error) {
-    console.error(error);
+    // handle specific error for `auth/account-exists-with-different-credential`
+    if (error.code === 'auth/account-exists-with-different-credential') {
+      // handle the error
+      STREET.notify.errorMessage(
+        `Cannot use Google login with your email, try using Microsoft login instead.`
+      );
+      console.log(
+        'Cannot use Google login with your email, try using Microsoft login instead.'
+      );
+    } else {
+      STREET.notify.errorMessage(
+        `Unexpected error using Google for login: ${error}.`
+      );
+      console.error(error);
+    }
   }
 };
 
-export { signIn };
+const signInMicrosoft = async () => {
+  try {
+    const provider = new OAuthProvider('microsoft.com');
+    const { user } = await signInWithPopup(auth, provider);
+    STREET.notify.successMessage(
+      `Successful login with Microsoft authentication.`
+    );
+    // first signIn to ga
+    if (user.metadata.creationTime !== user.metadata.lastSignInTime) return;
+    sendMetric('Auth', 'newAccountCreation');
+    posthog.capture('user_signed_up', {
```

Comment:
if you want to track posthog, you could add something like `provider: 'microsoft'` as a kv pair